packages/TorneloScoresheet/src/components/ChessBoard/ChessBoard.tsx

```diff
@@ -0,0 +1,36 @@
+import React from 'react';
+import { View } from 'react-native';
+import { BoardPosition } from '../../types/ChessMove';
+import ChessSquare from '../ChessSquare/ChessSquare';
+import RoundedView from '../RoundedView/RoundedView';
+import { styles } from './style';
+
+export type ChessBoardProps = {
+  boardPositions: BoardPosition[][];
```

Comment:
I think this will always be 8-by-8 yeah? We might want to make this a tuple of tuples. Always prefer the stricter type definition - in this case, the type doesn't specify how many `BoardPosition`s there are. Since we know this at compile time, we may as well encode it in the type.

```ts

type Row = [BoardPosition, BoardPosition, BoardPosition, BoardPosition, BoardPosition, BoardPosition, BoardPosition, BoardPosition];

type BoardPositions = [Row, Row, Row, Row, Row, Row, Row, Row];

```

(This can be written a bit cleaner/fancier, but requires some Typescript magic)